app/src/main/java/emu/skyline/AppDialog.kt

```diff
@@ -101,5 +82,14 @@ class AppDialog : BottomSheetDialogFragment() {
 
             shortcutManager.requestPinShortcut(info.build(), null)
         }
+
+        dialog?.setOnKeyListener { _, keyCode, event ->
+            if (keyCode == KeyEvent.KEYCODE_BUTTON_B && event.action == KeyEvent.ACTION_UP) {
+                dialog?.onBackPressed()
+                true
+            } else {
```

Comment:
Is this `else` really needed? It will be `false` anyway without it

Reply:
It's needed because of how returning from lambda works in Kotlin.